src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Command/AuthorizePlugin.php

```diff
@@ -23,6 +23,8 @@
 class AuthorizePlugin extends AbstractPlugin implements CommandByOrderInterface
 {
     /**
+     * {@inheritDoc}
+     *
```

Comment:
* - Sends payment authorize request to AfterPay gateway.
* - Saves the transaction result in Quote for future recognition.